Local project graph plugins fail to load preventing any tasks being run

<!-- Please do your best to fill out all of the sections below! -->

## Current Behavior
<!-- What is the behavior that currently you experience? -->
When registering a local project graph plugin in nx.json we now get an error in 14.8.2 when we try to run any tasks:

```json
"plugins": [
    "./tools/project-graph-plugins/plugin.js"
  ],
```

We now get a module not found error when trying to run any tasks. I believe this is because it is trying to resolve it from a project root, whereas before this was resolved based on the workspace root.

I believe this issue was introduced by the following commit:

https://github.com/nrwl/nx/commit/9b63ce167acb08faaf55bdf899055ae04a87036f#diff-c13565e7b12b241ee14541d4a40870cdc0484ae1bfdb5488be0767d3c0b8455fR53

## Expected Behavior
<!-- What is the behavior that you expect to happen? -->
<!-- Is this a regression? .i.e Did this used to be the behavior at one point?  -->
This should continue to work as it previously did, allowing relative paths to project graph plugins.

## Steps to Reproduce
<!-- Help us help you by making it easy for us to reproduce your issue! -->
See reproduction environment here:
https://github.com/nrwl/nx-examples/pull/233

<!-- Can you reproduce this on https://github.com/nrwl/nx-examples? -->
<!-- If so, open a PR with your changes and link it below. -->
<!-- If not, please provide a minimal Github repo -->
<!-- At the very least, provide as much detail as possible to help us reproduce the issue -->


### Failure Logs
<!-- Please include any relevant log snippets or files here. -->
```bash
Error: Cannot find module './tools/project-graph-plugins/plugin.js'
Require stack:
- /Users/ashley/Documents/GitHub/ux-aspects-reuse-poc/node_modules/nx/src/utils/nx-plugin.js
- /Users/ashley/Documents/GitHub/ux-aspects-reuse-poc/node_modules/nx/src/config/workspaces.js
```

### Environment
<!-- It's important for us to know the context in which you experience this behavior! -->
<!-- Please paste the result of `nx report` below! -->

```
   Node : 16.13.0
   OS   : darwin x64
   yarn : 1.22.19
   
   nx : 14.8.2
   @nrwl/angular : 14.8.2
   @nrwl/cypress : 14.8.2
   @nrwl/detox : Not Found
   @nrwl/devkit : 14.8.2
   @nrwl/esbuild : Not Found
   @nrwl/eslint-plugin-nx : 14.8.2
   @nrwl/expo : Not Found
   @nrwl/express : Not Found
   @nrwl/jest : 14.8.2
   @nrwl/js : 14.8.2
   @nrwl/linter : 14.8.2
   @nrwl/nest : Not Found
   @nrwl/next : Not Found
   @nrwl/node : Not Found
   @nrwl/nx-cloud : 14.7.0
   @nrwl/nx-plugin : Not Found
   @nrwl/react : 14.8.2
   @nrwl/react-native : Not Found
   @nrwl/rollup : 14.8.2
   @nrwl/schematics : Not Found
   @nrwl/storybook : 14.8.2
   @nrwl/web : 14.8.2
   @nrwl/webpack : 14.8.2
   @nrwl/workspace : 14.8.2
   typescript : 4.8.4
   ---------------------------------------
   Local workspace plugins:
   ---------------------------------------
   Community plugins:
         @ngrx/component-store: 14.0.2
         @ngrx/effects: 14.0.2
         @ngrx/entity: 14.0.2
         @ngrx/router-store: 14.0.2
         @ngrx/store: 14.0.2
         @ngrx/store-devtools: 14.0.2
```

--- a/packages/nx/src/project-graph/build-nodes/workspace-projects.ts
+++ b/packages/nx/src/project-graph/build-nodes/workspace-projects.ts
@@ -50,7 +50,7 @@ export function buildWorkspaceProjectNodes(
     p.targets = mergePluginTargetsWithNxTargets(
       p.root,
       p.targets,
-      loadNxPlugins(ctx.workspace.plugins, [p.root], p.root)
+      loadNxPlugins(ctx.workspace.plugins)
     );
 
     const projectType =

